strings.txt

```diff
@@ -16858,3 +16858,955 @@
+    cs = Stáhnout pomocí připojení přes mobilní síť?
+    it = Vuoi scaricare utilizzando una connessione di rete cellulare?
+    nb = Last ned via en mobil nettverkskobling?
+    zh-Hant = 用手機網絡連接下載嗎？
+    zh-Hans = 用手机网络连接下载吗？
+    th = ดาวน์โหลดโดยใช้การเชื่อมต่อเครือข่ายมือถือ?
+    ja = 携帯通信ネットワークで接続してダウンロードしますか？
+    ro = Descărcați utilizând o conexiune prin rețeaua de telefonie mobilă?
+    ar = تنزيل باستخدام اتصال الشبكة الخلوية؟
+    sk = Sťahovať prostredníctvom pripojenia na mobilnú sieť?
+    es = ¿Descargar con conexión de red móvil?
+    pl = Czy pobrać, używając połączenia z siecią komórkową?
+    nl = Downloaden via een mobiele gegevensverbinding?
+    pt = Baixar usando uma conexão de rede celular?
+
+  [download_over_mobile_message]
+    comment = Текст ошибки при загрузке через сотовую связь
+    en = This could be considerably expensive with some plans or if roaming.
+    ru = На некоторых тарифных планах или в роуминге это может привести к значительным расходам.
+    fr = Cela pourrait être très cher avec certains abonnements ou si vous êtes en déplacement.
+    da = Det kan blive temmelig dyrt med nogle abonnementer eller via roaming.
+    id = Ini bisa menjadi jauh mahal pada beberapa paket atau jika roaming.
+    ko = 이는 일부 플랜이나 로밍할 경우에 비싸다고 간주될 수 있습니다.
+    sv = Detta kan vara mycket dyrt med vissa abonnemang och vid roaming.
+    tr = Bu işlem bazı planlarla ya da dolaşım ise büyük ölçüde pahalı olabilir.
+    uk = Це може бути задорого за деякими планами або за умовами роумінгу.
+    vi = Như vậy sẽ rất đắt tiền với một số gói dữ liệu hoặc khi chuyển vùng.
+    hu = Ez jelentősen drága lehet némely előfizetés vagy roaming keretein belül.
+    de = Das könnte mit einigen Tarifen oder beim Roaming sehr teuer werden.
+    fi = Tämä vaihtoehto saattaa olla huomattavasti kalliimpi tietyillä sopimuksilla tai roaming-yhteydellä.
+    cs = Toto by mohli být s některými tarify nebo roamingem výrazně dražší.
+    it = Quest'operazione potrebbe essere piuttosto costosa con alcuni piani o in roaming.
+    nb = Dette kan bli betraktelig dyrere med enkelte abonnement, eller ved bruk av roaming.
+    zh-Hant = 用一些方案或漫遊的話這可能相當昂貴。
+    zh-Hans = 用一些方案或漫游的话这可能相当昂贵。
+    th = มันอาจมีราคาสูงมากหากใช้แผนโทรศัพท์บางประเภทหรือหากทำการโรมมิ่ง
+    ja = プランによって、またはローミングしている場合、非常に高額になる可能性があります。
+    ro = Aceasta poate fi destul de costisitoare în cazul unor abonamente sau dacă sunteți pe roaming.
+    ar = قد يكون هذا مكلفا جدا لبعض الخطط أو عند التجوال.
+    sk = V prípade niektorých plánov alebo použitím roamingu by to mohlo byť značne nákladné.
+    es = Podría ser muy caro con ciertos planes o con itinerancia de datos.
+    pl = Może to być kosztowne przy niektórych planach taryfowych lub w roamingu.
+    nl = Met sommige abonnementen of bij roaming kan dit behoorlijk duur zijn.
+    pt = Isto pode ser significativamente caro, com alguns planos ou se roaming.
+
+  [location_services_disabled_header]
```

Comment:
И enable_location_services
Надо вычитывать весь файл и удалять лишние переводы.
> On Apr 26, 2016, at 4:09 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61081433>:
> 
> > +    cs = Toto by mohli být s některými tarify nebo roamingem výrazně dražší.
> > +    it = Quest'operazione potrebbe essere piuttosto costosa con alcuni piani o in roaming.
> > +    nb = Dette kan bli betraktelig dyrere med enkelte abonnement, eller ved bruk av roaming.
> > +    zh-Hant = 用一些方案或漫遊的話這可能相當昂貴。
> > +    zh-Hans = 用一些方案或漫游的话这可能相当昂贵。
> > +    th = มันอาจมีราคาสูงมากหากใช้แผนโทรศัพท์บางประเภทหรือหากทำการโรมมิ่ง
> > +    ja = プランによって、またはローミングしている場合、非常に高額になる可能性があります。
> > +    ro = Aceasta poate fi destul de costisitoare în cazul unor abonamente sau dacă sunteți pe roaming.
> > +    ar = قد يكون هذا مكلفا جدا لبعض الخطط أو عند التجوال.
> > +    sk = V prípade niektorých plánov alebo použitím roamingu by to mohlo byť značne nákladné.
> > +    es = Podría ser muy caro con ciertos planes o con itinerancia de datos.
> > +    pl = Może to być kosztowne przy niektórych planach taryfowych lub w roamingu.
> > +    nl = Met sommige abonnementen of bij roaming kan dit behoorlijk duur zijn.
> > +    pt = Isto pode ser significativamente caro, com alguns planos ou se roaming.
> > +
> > +  [location_services_disabled_header]
> Уже есть [location_is_disabled_long_text] [gps_is_disabled_long_text]
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61081433>